components/02-elements/breadcrumbs/breadcrumbs.hbs

```diff
@@ -5,9 +5,11 @@
                 {{ this }}
             </span>
         {{else}}
-            <a href="#" class="breadcrumbs__item">
-                {{ this }}
-            </a>
+            <span class="breadcrumbs__item">
+                <a href="#" class="breadcrumbs__link">
```

Comment:
set a href as a context element in config, please
add also attribute `aria-label` with text setting in config file (in context)

Reply:
Done